.changeset/tiny-pigs-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/badge': minor
+---
+
+Remove neutral tone from IndicatorDot
```

Comment:
It looks like the tone prop has been removed altogether. It's probably worth mentioning that here 🙂

Reply:
Great idea!